README.md

```diff
@@ -163,6 +163,8 @@ In case you need to run a specific suite of tests you can execute
 - [Lightstep](https://github.com/lightstep/opentelemetry-demo)
 - [New Relic](https://github.com/newrelic/opentelemetry-demo)
 
+See [Forking](./docs/forking.md) for more information about creating your own fork.
```

Comment:
Minor, maybe add some words here to encourage folks to add their forks here.

Reply:
Added here: https://github.com/open-telemetry/opentelemetry-demo/pull/447/files#diff-1c2caaad726b38f555fa598c8c26fbec6f8a81c8793ce0074a280bf8f375d744R9